tests/test_gis_models.py

```diff
@@ -0,0 +1,273 @@
+import os
+
+from django.contrib.gis.geos import Polygon, MultiPolygon, Point, LineString
+
+import pytest
+
+from asset_dashboard import models
+
+
+def get_all(model):
+    """Helper function since every test gets a queryset.
+    Also that the qs isn't empty."""
+    if bool(os.environ.get('TEST_GIS')):
```

Comment:
This would be more appropriate to implement in a `skipif` mark: https://docs.pytest.org/en/6.2.x/reference.html#pytest-mark-skipif

Reply:
Cool, I added that here: f8c2408

I combined the pytest decorators into a single decorator since they're used for every test and both pertain to the gis database. Much cleaner than my original implementation. Let me know if you see any room for improvement.

Reply:
Thanks to this SO: https://stackoverflow.com/a/44635908